### PR TITLE
Add BGS Escape Creation

### PR DESCRIPTION
**Pull request by DJLegends1011**
[Escape Creation](https://creations.bethesda.net/en/starfield/details/1106af41-fac5-431e-aa1a-163b3e5d2892/Escape)

### CODE REVIEW

**Comment by pStyl3** [comment]
Thanks for the PR. Please move the new entry inbetween `sfbgs00a_j.esm` and `sfbgs00e.esm`

**Comment by DJLegends1011** [comment]
will do!

**Comment by DJLegends1011** [comment]
and finished

File: masterlist.yaml [hunk 1]
```diff
@@ -396,6 +396,12 @@ plugins:
         name: 'Blackout Kodama Skin'
     group: *bgsCreationsGroup
 
+  - name: 'sfbgs00b.esm'
+    url:
+      - link: 'https://creations.bethesda.net/en/starfield/details/1106af41-fac5-431e-aa1a-163b3e5d2892/Escape/'
+        name: 'Escape'
+    group: *bgsCreationsGroup
+    
   - name: 'sfbgs00e.esm'
```
Comment (sibir-ine) [inline masterlist.yaml]: There are some extra spaces here:

```suggestion
    group: *bgsCreationsGroup

  - name: 'sfbgs00e.esm'
```

File: masterlist.yaml [hunk 2]
```diff
@@ -438,6 +444,8 @@ plugins:
         name: 'Trackers Alliance: The Vulture'
     group: *bgsCreationsGroup
 
+
+
 ###### Verified Creations ######
```
Comment (sibir-ine) [inline masterlist.yaml]: There are some extra newlines here:

```suggestion

###### Verified Creations ######
```

**Review by sibir-ine — CHANGES_REQUESTED** [review]

**Review by sibir-ine — APPROVED** [review]
Thanks!

**Pull request merged**